UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -65,6 +67,84 @@ void InitAdminController()
 		LoadBanList();
 		LoadCurrentAdmins();
 		LoadWhiteList();
+		AdminLogStart();
+	}
+
+	private void AdminLogStart()
+	{
+		adminLogPath = Path.Combine(Application.streamingAssetsPath, "admin", "adminlog1.txt");
+		adminLog2Path = Path.Combine(Application.streamingAssetsPath, "admin", "adminlog2.txt");
+
+		if (!File.Exists(adminLogPath))
+		{
+			//Creates file
+			var file = File.CreateText(adminLogPath);
+
+			//Todo add to file round number here
+			file.WriteLine("true");
+
+			file.Close();
+		}
+
+		if (!File.Exists(adminLog2Path))
+		{
+			//Creates file
+			var file = File.CreateText(adminLog2Path);
+
+			//Todo add to file round number here
+			file.WriteLine("false");
+
+			file.Close();
+		}
+
+		var first = File.ReadLines(adminLogPath).First() == "true";
+
+		var second = File.ReadLines(adminLog2Path).First() == "true";
+
+		if (first && !second)
```

Comment:
comments are great, but if you're going to add comments it would be more helpful to explain "why" rather than "what"...which leads me to my next question, why is it doing this check?

Reply:
Thats all part of it deciding which file to write to next.

Heres a pic of what i had to do:

![image](https://user-images.githubusercontent.com/56727168/81172860-f8c8d880-8f96-11ea-9ed8-8c54efc02045.png)
The number in the circle is which file it will use